packages/foreman/foreman-proxy/foreman-proxy.spec

```diff
@@ -4,20 +4,18 @@
 %global scl_ruby_bin /usr/bin/%{?scl:%{scl_prefix}}ruby
 %global scl_rake /usr/bin/%{?scl:%{scl_prefix}}rake
 
-# set and uncomment all three to set alpha tag
-#global alphatag RC1
-#global dotalphatag .%{alphatag}
-#global dashalphatag -%{alphatag}
+%global prerelease develop
+%global release 1
 
 Name:           foreman-proxy
 Version:        1.20.0
-Release:        0.develop%{?dotalphatag}%{?dist}
+Release:        %{?prerelease:0.}%{release}%{?prerelease:.}%{?prerelease}%{?dist}
 Summary:        Restful Proxy for DNS, DHCP, TFTP, PuppetCA and Puppet
 
 Group:          Applications/System
 License:        GPLv3+
 URL:            https://theforeman.org/projects/smart-proxy
-Source0:        https://downloads.theforeman.org/%{name}/%{name}-%{version}%{?dashalphatag}.tar.bz2
+Source0:        https://downloads.theforeman.org/%{name}/%{name}-%{version}-%{prerelease}.tar.bz2
```

Comment:
This could also use the prerelease macro instead of hardcoding. Probably left over debug code :)